CHANGELOG.md

```diff
@@ -5,11 +5,16 @@
 ### Added
 - Add `vendor[1,2,3]-prompt` as options to ConnectionConfig-CT in `tcd_latest.xsd`.
 
+- Add JDBC Kerberos scenario for postgres 
```

Comment:
Also released

Reply:
This seems to be added in 2020.4 , [MasterBranch](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/scenarios) vs [2020.4Branch](https://github.com/tableau/connector-plugin-sdk/tree/dev-2020.4/samples/scenarios). Seems i should keep this line ?

Reply:
Each of these that were previously in Unreleased should be moved to dated entries below.

Reply:
Oh yes , fixed this and the previous two as well. Put them under the specified released date